src/components/ConversationSettings/LockingSettings.vue

```diff
@@ -25,26 +25,40 @@
 			{{ t('spreed', 'Locking the conversation prevents anyone to post messages or start calls.') }}
 		</div>
 		<div>
-			<input id="moderation_settings_lock_conversation_checkbox"
-				aria-describedby="moderation_settings_lock_conversation_hint"
-				type="checkbox"
-				class="checkbox"
-				name="moderation_settings_lock_conversation_checkbox"
-				:checked="isReadOnly"
-				:disabled="isReadOnlyStateLoading"
-				@change="toggleReadOnly">
-			<label for="moderation_settings_lock_conversation_checkbox">{{ t('spreed', 'Lock conversation') }}</label>
+			<h3>{{ t('spreed', 'Limit writing to a conversation') }}</h3>
+			choosen: {{ writingConversationSelected }}
+			<p>
+				<Multiselect id="limit_writing"
+					v-model="writingConversationSelected"
+					:options="writingConversationOptions"
+					:placeholder="t('spreed', 'Limit writing to conversations')"
+					label="label"
+					track-by="value"
+					:disabled="isReadOnlyStateLoading"
+					@input="saveWritingConversations" />
+			</p>
 		</div>
 	</div>
 </template>
 
 <script>
 import { showError, showSuccess } from '@nextcloud/dialogs'
 import { CONVERSATION } from '../../constants'
+import Multiselect from '@nextcloud/vue/dist/Components/Multiselect'
+
+const writingConversationOptions = [
+	{ value: 0, label: t('spreed', 'Everyone can read and write') },
+	{ value: 1, label: t('spreed', 'Lock conversation') },
+	{ value: 2, label: t('spreed', 'Only moderators can write') },
```

Comment:
```suggestion
	{ value: 0, label: t('spreed', 'Everyone can read and write') },
	{ value: 2, label: t('spreed', 'Only moderators can write') },
	{ value: 1, label: t('spreed', 'Lock conversation') },
```

Let's change the order to something logical (not the value)